internal/cli/kraft/cloud/certificate/create/create.go

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kraftcloud "sdk.kraft.cloud"
+	kccertificates "sdk.kraft.cloud/certificates"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+	"kraftkit.sh/iostreams"
+	"kraftkit.sh/log"
+)
+
+type CreateOptions struct {
+	Auth   *config.AuthConfig                 `noattribute:"true"`
+	Client kccertificates.CertificatesService `noattribute:"true"`
+	Metro  string                             `noattribute:"true"`
+	Chain  string                             `local:"true" long:"chain" short:"h" usage:"The chain of the certificate"`
```

Comment:
`-h` is already used via the help menu and I receive a panic.

Let's use the following instead:

```diff
-       Chain  string                             `local:"true" long:"chain" short:"h" usage:"The chain of the certificate"`
-       CName  string                             `local:"true" long:"cname" short:"c" usage:"The common name of the certificate"`
+       Chain  string                             `local:"true" long:"chain" short:"C" usage:"The chain of the certificate"`
+       CN     string                             `local:"true" long:"cn" short:"c" 
```